### PR TITLE
refactor(utils): use hubble to get user token

### PR DESCRIPTION
**Pull request by npitsillos**
This PR refactor's this [function](https://github.com/jina-ai/now/blob/main/now/utils.py#L127) to use `hubble.utils.auth.Auth.get_user_token()` for obtaining the `auth_token` without requiring the file to be created in the GHA workflow.

---

- [ ] This PR references an open issue
- [ ] Test added
- [ ] Documentation adjusted

### CODE REVIEW

**Review by azayz — APPROVED** [review]
LGTM, wait for CI to work

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -260,10 +265,10 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           S3_CUSTOM_DATA_PATH: ${{ secrets.S3_CUSTOM_DATA_PATH }}
-#      - name: Setup tmate session for debugging
-#        if: ${{ failure() || (github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled) }}
-#        uses: mxschmitt/action-tmate@v3
-#        timeout-minutes: 60
+      - name: Setup tmate session for debugging
```
Comment (makram93) [inline .github/workflows/ci.yml:268]: Let's comment out this part. One can enable it for their respective PR if needed but having this, in general, is not a wanted behavior

File: now/utils.py [hunk 2]
```diff
@@ -125,9 +125,7 @@ def jina_auth_login():
 
 
 def get_info_hubble(user_input):
-    with open(user('~/.jina/config.json')) as fp:
-        config_val = json.load(fp)
-        user_token = config_val['auth_token']
+    user_token = hubble.utils.auth.Auth.get_auth_token()
```
Comment (nomagick) [inline now/utils.py:128]:
```suggestion
```

File: now/utils.py [hunk 3]
```diff
@@ -125,9 +125,7 @@ def jina_auth_login():
...
     client = hubble.Client(token=user_token, max_retries=None, jsonify=True)
```
Comment (nomagick) [inline now/utils.py:129]:
```suggestion
    client = hubble.Client(max_retries=None, jsonify=True)
```
Just omit the token parameter will be enough.

**Review by nomagick — COMMENTED** [review]
You don't have to set the token manually.
The SDK is capable of finding the token by itself.

**Review by makram93 — APPROVED** [review]
LGTM! Just a minor change - disable tmux

File: now/utils.py (hunk 3 above)
Comment (npitsillos) [reply now/utils.py:129]: Thanks for pointing that out! I didn't know that, I will make this change.

Comment (nomagick) [reply now/utils.py:129]: To get the token `Auth.get_auth_token()`.
Or `client._token`.

**Pull request merged**